src/api/test.rs

```diff
@@ -976,84 +945,84 @@ fn output_frameno_incremental_reorder_minus(missing: u64) {
-          (9, true), // P-frame
-          (7, true), // B0-frame
-          (6, true), // B1-frame (first)
-          (7, true), // B0-frame (show existing)
-          (8, true), // B1-frame (second)
-          (9, true), // P-frame (show existing)
+          (9, true),
+          (7, true),
+          (6, true),
+          (7, true),
+          (8, true),
+          (9, true),
         ][..]
       }
       1 => {
         &[
-          (0, true),  // I-frame
-          (4, true),  // P-frame
-          (2, true),  // B0-frame
-          (1, true),  // B1-frame (first)
-          (2, true),  // B0-frame (show existing)
-          (3, true),  // B1-frame (second)
-          (4, true),  // P-frame (show existing)
-          (5, true),  // I-frame
-          (5, false), // Last frame (missing)
-          (7, true),  // B0-frame
-          (6, true),  // B1-frame (first)
-          (7, true),  // B0-frame (show existing)
-          (8, true),  // B1-frame (second)
-          (8, false), // Last frame (missing)
+          (0, true), // I-frame
+          (4, true),
+          (2, true),
+          (1, true),
+          (2, true),
+          (3, true),
+          (4, true),
+          (5, true), // I-frame
+          (5, false),
+          (7, true),
+          (6, true),
+          (7, true),
+          (8, true),
+          (8, false),
         ][..]
       }
       2 => {
         &[
-          (0, true),  // I-frame
-          (4, true),  // P-frame
-          (2, true),  // B0-frame
-          (1, true),  // B1-frame (first)
-          (2, true),  // B0-frame (show existing)
-          (3, true),  // B1-frame (second)
-          (4, true),  // P-frame (show existing)
-          (5, true),  // I-frame
-          (5, false), // Last frame (missing)
-          (7, true),  // B0-frame
-          (6, true),  // B1-frame (first)
-          (7, true),  // B0-frame (show existing)
+          (0, true), // I-frame
+          (4, true),
+          (2, true),
+          (1, true),
+          (2, true),
+          (3, true),
+          (4, true),
+          (5, true), // I-frame
+          (5, false),
+          (7, true),
+          (6, true),
+          (7, true),
           (7, false), // 2nd last (missing)
-          (7, false), // Last frame (missing)
+          (7, false),
         ][..]
       }
       3 => {
         &[
-          (0, true),  // I-frame
-          (4, true),  // P-frame
-          (2, true),  // B0-frame
-          (1, true),  // B1-frame (first)
-          (2, true),  // B0-frame (show existing)
-          (3, true),  // B1-frame (second)
-          (4, true),  // P-frame (show existing)
-          (5, true),  // I-frame
-          (5, false), // Last frame (missing)
+          (0, true), // I-frame
+          (4, true),
+          (2, true),
+          (1, true),
+          (2, true),
+          (3, true),
+          (4, true),
+          (5, true), // I-frame
+          (5, false),
           (5, false), // 3rd last (missing)
```

Comment:
did you maybe want to drop these comments as well?

Reply:
Good catch, thanks. I meant to drop all of them except the "I-frame" comments.